omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -151,5 +151,9 @@ def ldap_authentication_enabled?
     node['private_chef'].attribute?('ldap') &&
       !(node['private_chef']['ldap'].nil? || node['private_chef']['ldap'].empty?)
   end
+
+  def is_backend?()
```

Comment:
I believe we have a function like this in enterprise-chef-common?  Could we consider using that one instead?